inst/include/dust/interface.hpp

```diff
@@ -229,8 +266,11 @@ inline std::vector<size_t> validate_size(cpp11::sexp r_x, const char * name) {
   return x;
 }
 
-inline std::vector<size_t> r_index_to_index(cpp11::sexp r_index,
-                                            size_t nmax) {
+inline std::vector<size_t> r_index_to_index(cpp11::sexp r_index, size_t nmax) {
```

Comment:
This and following function have confusing names, and also I cannot immediately work out their purpose - can we add a comment above both perhaps with their intended use?

Reply:
sure, added that in 5b3cea7